.storybook/main.ts

```diff
@@ -0,0 +1,17 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    '@storybook/addon-onboarding',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
```

Comment:
we don't have this in the anatomy-react storybook/main.ts file, is it something we should add?

Reply:
Chromatic, you mean? It gets installed automatically with newer versions of storybook I believe, but this package won't need it. I'll clean this up as well when I clean up the assets folder.